main.tf

```
@@ -3,6 +3,7 @@ module "state_bucket" {
   bucket        = var.backend_config.bucket
   force_destroy = var.backend_config.force_destroy
   versioning    = var.backend_config.versioning
+  tags          = var.tags
```

Should be `var.backend_config.tags`.

```suggestion
  tags          = var.backend_config.tags
```

---

Updated